docs/cas-server-documentation/installation/Audits.md

```diff
@@ -23,6 +23,25 @@ The following endpoints are provided by CAS:
 |--------------------------|------------------------------------------------
 | `auditLog`               | Provides a JSON representation of all the audit log.
 
+Navigating to `/actuator/auditLog` displays the complete audit for the past number of days determined by `cas.audit.numberOfDaysInHistory`(default is 30 days);
+
+You can specify an interval of log entries to return by adding a Duration Syntax to the navigated path.  This interval will be subtracted from the current 
+date and time when the query is executed.  For instance `/actuator/auditLog/PT1H` will return only entries for the past our.  An additional path can be added to 
+limit the types of entries that will be returned (i.e `/actuator/auditLog/PTIH/SERVICE_TICKET_CREATED`).
```

Comment:
These should not be selectors. There are query parameters.

Reply:
They don't work as query params though, they do as path selectors.

Reply:
Why not?

Reply:
Sorry, it was my misunderstanding.